js/planning.js

```diff
@@ -324,16 +378,227 @@ var GLPIPlanning = {
          );
 
       $('#refresh_planning').click(function() {
-         calendar.refetchEvents();
+         GLPIPlanning.refresh();
       });
 
       // attach the date picker to planning
       GLPIPlanning.initFCDatePicker();
    },
 
+   refresh: function() {
+      GLPIPlanning.calendar.refetchResources();
```

Comment:
Function does not exists if resource plugin is not loaded.
```suggestion
      if (typeof(GLPIPlanning.calendar.refetchResources) == 'function') {
         GLPIPlanning.calendar.refetchResources();
      }
```